code_challenges.rb

```diff
@@ -7,42 +7,84 @@
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
 
-
-
-
+# create a method that takes in an array.
+#filter through the array, return only odd numbers 
+#select looks for true values
+# getting error that % is an undefined method?????
+
+
+# .odd? .even? doesn't work, but on the lecture notes when we learned it, it does.
+
+# Just this first challenge took me all of saturday to do.
+def odds_only array
+    # create a variable to store odd numbers
+    odds = []
+    # iterate through every element of the array and 
+    array.each do |value|
+        # check if the value is an integer,
+        if value.is_a?(Integer)
+            # If it is, check if it's odd. If it is, shovel it into the odds array
+            if value % 2 == 1
+                odds << value
+            end
+        end
+    end
+    # ruby has implicit return so I put odds. sort as the last executed code to return that.
+    odds.sort
+end
+
+# puts odds_only fullArr2
+# puts odds_only fullArr1
```

Comment:
This works, but I would pay attention to what Ruby has returned here, it seems to have returned back 3 different integers, not inside of an array like the expected output is supposed to return.